src/site/_data/courses/design/meta.yml

```diff
@@ -6,7 +6,7 @@
 # on or before this date will not receive the "New" badge.
 date: 2021-11-03
 url: /learn/design/
-draft: true
+draft: false
```

Comment:
I'd probably just remove these lines rather than set `false`. Up to you.

Reply:
The css course has kept it, so i keep them in for consistency (and im afraid to break something i don't know about)